x-pack/plugins/elastic_assistant/server/lib/data_stream/documents_data_writer.ts

```diff
@@ -73,8 +76,8 @@ export class DocumentsDataWriter implements DocumentsDataWriter {
           body: await this.buildBulkOperations(params),
         },
         {
-          // Increasing timout to 2min as KB docs were failing to load after 30s
-          requestTimeout: 120000,
+          // Increasing timout to 10min as KB docs were failing to load after 30s
+          requestTimeout: 600000,
```

Comment:
Wondering if there's any risk of this happening, where the browser retries the request after 2 minutes: https://github.com/elastic/kibana/pull/194166/commits/513d4bf5e4c35bc122af53f8b69e6adbe0c7dd77#diff-fd04f2600e1be329a4f61e60e32536a70b33742a24caffeb1a03101510dc0b10R18